README.md

```diff
@@ -0,0 +1,95 @@
+
+# 3D City Database User Manual
+
+The 3DCityDB User Manual is built and hosted on [ReadTheDocs.org](https://readthedocs.org).
+Please use the link below to find the online version of this documentation.
+A PDF version is available here: <https://www.3dcitydb.org/3dcitydb/documentation/>
```

Comment:
The referenced PDF is outdated and is not continued anymore. Thus, I suggest to either remove this sentence or put into an extra section "Documentation for previous releases" at the end of the page.